dev/com.ibm.ws.webcontainer/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -162,7 +162,7 @@ servletDestroyWaitTime.desc=Wait time in seconds for an active request to comple
 servletPathForDefaultMapping.name=Set the servlet path value for a default mapping
 servletPathForDefaultMapping.desc=Set the servlet path value to the request URI minus the context path. The path information is null when a servlet is used as a default mapping. The default value is true for version 4.0 or later of the servlet feature. It is false for other servlet features. When mapping is to the /* pattern, the servlet path is empty and the path information starts with a leading slash (/).
 
-throwExceptionWhenUnableToCompleteOrDispatch.name=Throw exception for not complete or dispatch asyn
+throwExceptionWhenUnableToCompleteOrDispatch.name=Throw exception for not complete or dispatch async
```

Comment:
```suggestion
throwExceptionWhenUnableToCompleteOrDispatch.name=Throw an exception for an incomplete or dispatch async request.
```

Reply:
Outdated to to my push, but I made a new suggestion further down.